src/components/category/Category.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { categories } from "../../backend/db/categories";
```

Comment:
use api call for instead of importing db component

Reply:
Used it in the next PR. So not changing here.